fe/fe-core/src/main/java/org/apache/doris/catalog/Catalog.java

```diff
@@ -5092,6 +5098,35 @@ public void renameTable(Database db, OlapTable table, TableRenameClause tableRen
         LOG.info("rename table[{}] to {}", tableName, newTableName);
     }
 
+    public void renameTable(Database db, Table table, TableRenameClause tableRenameClause) throws DdlException {
```

Comment:
This method is just same as `renameTable` above?

Reply:
yes, but the param is different. one is the `olaptable`, this is `table`